commit 0964697dd611ddda8963d2c9c2d7977bf969f990
Author: Maciej S. Szmigiero <mail@maciej.szmigiero.name>
Date:   Wed Jan 18 21:31:11 2017 +0100

    w1: ds2490: USB transfer buffers need to be DMAable
    
    commit 61cd1b4cd1e8f7f7642ab64529d9bd52e8374641 upstream.
    
    ds2490 driver was doing USB transfers from / to buffers on a stack.
    This is not permitted and made the driver non-working with vmapped stacks.
    
    Since all these transfers are done under the same bus_mutex lock we can
    simply use shared buffers in a device private structure for two most common
    of them.
    
    While we are at it, let's also fix a comparison between int and size_t in
    ds9490r_search() which made the driver spin in this function if state
    register get requests were failing.
    
    Signed-off-by: Maciej S. Szmigiero <mail@maciej.szmigiero.name>
    Acked-by: Evgeniy Polyakov <zbr@ioremap.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1307_0964697dd611ddda8963d2c9c2d7977bf969f990.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1307_0964697dd611ddda8963d2c9c2d7977bf969f990.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
